src/hexagon.variables.css

```diff
@@ -1085,14 +1043,12 @@
   --titlebar-title-font-size: 1.7em;
   --titlebar-title-text-color: var(--theme-accent-color);
 
-
   /* Toggle */
-  --toggle-background-off-color: #D5D5D5;
-  --toggle-background-on-color: #D5D5D5;
-  --toggle-toggle-off-color: #ABABAB;
+  --toggle-background-off-color: #d5d5d5;
+  --toggle-background-on-color: #d5d5d5;
+  --toggle-toggle-off-color: #ababab;
```

Comment:
Not sure why most of this file has been changed in this PR, most of the changes don't affect the output and removing the additional newlines between sections makes it more difficult to read

It's fine to make these kind of changes, just generally they are done in their own PR